hack/e2e/eksctl.sh

```diff
@@ -69,13 +70,14 @@ function eksctl_create_cluster() {
     ${BIN} create nodegroup \
       --managed=false \
```

Comment:
`eksctl` has support for managed Windows nodegroups now (https://github.com/weaveworks/eksctl/releases/tag/v0.124.0), probably worth moving over?

Reply:
✅